### PR TITLE
[stable/concourse] Concourse v5.6.0

### PR DESCRIPTION
**Pull request by jamieklassen**
#### What this PR does / why we need it:
* upgrades Concourse to v5.6.0
* adds new configuration parameters for v5.6.0

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by pivotal-jamie-klassen** [comment]
@cirocosta @YoussB ready for review!

File: stable/concourse/values.yaml [hunk 1]
```diff
@@ -65,6 +65,19 @@ concourse:
...
     enableGlobalResources: true
 
+    ## The Future™ of resource checking.
+    ##
+    enableLidar: false
+
+    ## Interval on which the resource scanner will run to see if new checks need
+    ## to be scheduled.
+    ##
+    lidarScannerInterval: 1m
```
Comment (cirocosta) [inline stable/concourse/values.yaml]: Hey, 

I know that in various places we have values set in the yaml, but I'd rather not have these here - with an empty value, we'd then default to whatever we set as default in Concourse, which gives us less trouble if we end up changing defaults in Concourse itself 🤔  

Would you mind changing these to be empty?

Thanks!

**Review by cirocosta — CHANGES_REQUESTED** [review]
Thansk for the PR!

It'd be nice to not have those default values in `values.yaml`, and by the way, I noticed that you missed updating the version in the README.md 😁  

thanks!

File: stable/concourse/values.yaml (hunk 1 above)
Comment (jamieklassen) [reply stable/concourse/values.yaml]: done

**Pull request merged**